implementation/.github/workflows/push-buildpackage-and-send-dependency-update.yml

```diff
@@ -1,21 +1,39 @@
-# This workflow notifies language families on release of impl cnbs
-# Requires the following:
-# - secret PAKETO_BOT_GITHUB_TOKEN -> token with permissions to send dispatch to language family repos
-# - file .github/language-family-membership listing target <org>/<repo> per line
```

Comment:
@joshzarrabi Don't we need to still include these comments so that whoever is creating an implementation buildpack remembers to add the language-family-membership file and the dispatch secret?

Reply:
Yup! Thanks